code/modules/maps/tether/levels/tether_turfs.dm

```diff
@@ -100,6 +100,7 @@ VIRGO3B_TURF_CREATE(/turf/simulated/mineral/floor)
 	if(mineral || ignore_mapgen)
 		return
 	var/mineral_name
+	outdoors = FALSE
```

Comment:
Why is this indoors?

Reply:
Probably un-needed since the correction of mineral turfs being set to outdoors